src/main/java/com/gitblit/models/UserModel.java

```diff
@@ -574,6 +574,10 @@ public boolean canCreate(String repository) {
 			// admins can create any repository
 			return true;
 		}
+        ProjectModel projectModel = GitBlitWebApp.get().projects().getProjectModel(repository);

```

Comment:
Please fix indentation.
No `null` check needed b/c is done in `canManage()`.

Optional: add `canManage(projectModel)` to previous `if` statement: `if (canAdmin() || canManage(projectModel))`

Reply:
thanks for the input